cruise.umple/src/Main_Code.ump

```diff
@@ -116,10 +118,16 @@ class UmpleConsoleMain
   depend java.util.Arrays;
   depend joptsimple.*;
   depend java.io.*;
+  depend java.text.DecimalFormat;
 
   UmpleConsoleConfig cfg;
 
   static OptionParser optparser = initializeOptionParser();
+  
+  //t0 repersents model run start time while t1 represents end time
+  Double t0 = 0.0;
+  Double t1 = 0.0;
+
```

Comment:
I would suggest changing names to ones which are more meaningful.
For example, `startTime` and `endTime`